docs/quick-tutorials/k8s-kafka-mapping.mdx

```diff
@@ -0,0 +1,166 @@
+---
+sidebar_position: 4
+title: Kafka topic-level access mapping
```

Comment:
This tutorial isn't updated. I suggest updating it as well.

Reply:
Updated